cabra/src/pages/components/TaskDetailsCard.tsx

```diff
@@ -70,7 +69,11 @@ export default function TaskDetails({ id }: Props) {
           {task.description || "<No description>"}
         </CardField>
         <CardField title="Subtasks" span={3}>
-          <SubtasksList parentTaskId={id} />
+          <RelatedTasksList
+            relationType={DirectedRelationType.Subtask}
+            parentTaskId={id}
+            tw="flex"
```

Comment:
I think you can move this 'flex' to the component

Reply:
Not really. The RelatedTaskList component is used in two ways: horizontally (subtasks) and vertically (other relations). I consider it better to generify it this way.